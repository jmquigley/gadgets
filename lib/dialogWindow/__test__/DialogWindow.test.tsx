'use strict';

import {
	getDefaultDialogWindowProps
} from '../';

test('Test retrieval of DialogWindow props object', () => {
	const props = getDefaultDialogWindowProps();

	expect(props).toBeTruthy();
	expect(props).toMatchSnapshot();
});
