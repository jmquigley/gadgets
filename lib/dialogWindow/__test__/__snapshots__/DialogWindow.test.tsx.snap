// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test retrieval of DialogWindow props object 1`] = `
Object {
  "bottom": "",
  "children": null,
  "className": "",
  "contentEditable": false,
  "controlled": true,
  "direction": "right",
  "disabled": false,
  "err": null,
  "errorMessage": "",
  "focus": false,
  "height": "",
  "icon": "bomb",
  "id": "",
  "left": "",
  "location": "none",
  "noedit": false,
  "nohover": false,
  "noripple": false,
  "obj": "DialogWindow",
  "onClose": [Function],
  "onOpen": [Function],
  "padding": "",
  "selected": false,
  "show": false,
  "sizing": "normal",
  "style": Object {},
  "testing": true,
  "theme": null,
  "title": "Dialog Window",
  "tooltip": "",
  "top": "",
  "visible": true,
  "width": "",
  "xcss": null,
}
`;
